infra/src/main/java/com/redhat/service/bridge/infra/models/dto/ConnectorStatus.java

```diff
@@ -0,0 +1,11 @@
+package com.redhat.service.bridge.infra.models.dto;
+
+// TODO-MC review statuses
```

Comment:
@lucamolteni Is there a reason why this status is independent from `BridgeStatus`? They look the same to me, so can they be combined (assuming we need to maintain our own status as per comment of @danielezonca)

Reply:
@robpblake I assumed MC will eventually have a different life cycles than our bridge instances, anyway when I review those if I discover they're identical we might consider using only one